doc/examples/HelloWorld.x

```diff
@@ -1,6 +1,6 @@
 module HelloWorld {
+    @Inject Console console;
     void run() {
```

Comment:
Is this an intended change for HelloWorld.x? If not, I guess it's better to leave as many master files as possible untouched. I also think that it makes more sense to inject a single dependency as close the usage point as possible, but I'll leave that to the original XTC language designers to determine if that is preferred or not.